index.html

```diff
@@ -4,11 +4,14 @@
         <link rel="stylesheet" href="css/canvas.css">
      </head>   
 <body>
-    <canvas id="myCanvas" height="500" width="1000"></canvas>
+    <canvas id="myCanvas" ></canvas>
+     <span class="tooltip"></span>
     <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.1.1/jquery.min.js"></script>
     <script src="js/parserManager.js"></script>
```

Comment:
Can be grouped to a single js file with gulp/grunt or any other tools(To be done later)